dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImplTest.java

```diff
@@ -389,7 +392,7 @@ public void activateDeactivateIndex () throws Exception {
 
         //***************************************************
         //Get the current indices
-        String liveActiveIndex = indexAPI.getActiveIndexName( ContentletIndexAPI.ES_LIVE_INDEX_NAME );
+        String liveActiveIndex = indexAPI.getActiveIndexName( IndexType.LIVE.getPrefix() );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'liveActiveIndex' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)